bridges/modules/relayers/src/lib.rs

```diff
@@ -235,6 +235,22 @@ pub mod pallet {
 	}
 
 	impl<T: Config<I>, I: 'static> Pallet<T, I> {
+		/// Relayers that have reserved some of their balance to get free priority boost
+		/// for their message delivery transactions.
+		pub fn registered_relayer(
+			relayer: &T::AccountId,
+		) -> Option<Registration<BlockNumberFor<T>, T::Balance>> {
+			RegisteredRelayers::<T, I>::get(relayer)
+		}
+
+		/// Map of the relayer => accumulated reward.
+		pub fn relayer_reward(
+			key1: <RelayerRewardsKeyProviderOf<T, I> as StorageDoubleMapKeyProvider>::Key1,
+			key2: <RelayerRewardsKeyProviderOf<T, I> as StorageDoubleMapKeyProvider>::Key2,
```

Comment:
let's try if this fixes the the clippy: https://github.com/paritytech/polkadot-sdk/actions/runs/13784874860/job/38550580066?pr=7120#step:4:27

```suggestion
			key1: &<RelayerRewardsKeyProviderOf<T, I> as StorageDoubleMapKeyProvider>::Key1,
			key2: &<RelayerRewardsKeyProviderOf<T, I> as StorageDoubleMapKeyProvider>::Key2,
```